ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -55,6 +55,36 @@ code change.
      - Bare-metal
      - \-
 
+Quality managed targets
```

Comment:
sounds more correct to say *quality-managed* (instead of *quality managed*)

Reply:
I have requested clarification from the relevant persons who approved this wording (Quality managed) to see if they meant your suggestion.

Reply:
The relevant persons have rejected this suggestion, "Quality Managed" is the approved term.